### DISCUSSION TITLE
v2.9.1 (fix)

### DISCUSSION THREAD

**Discussion by xoureldeen** [xoureldeen/Vectras-VM-Android#82] [Ideas]
- fixes and improvments [(endless booting)](https://github.com/epicstudios856/Vectras-VM-Android/issues/79)
- we are removed cpu and gpu defaults and options to avoid bugs (temporarily)

<hr /><em>This discussion was created from the release <a href='https://github.com/epicstudios856/Vectras-VM-Android/releases/tag/v2.9.1'>v2.9.1 (fix)</a>.</em>

**Comment by ommprakashswain** [comment]
Here the VM should be connected to the mobile device's internal storage, other wise in VM if there was no inter-net connection I can't be able to import file into the VM .

**Reply by xoureldeen** [reply]
use iso craft

**Comment by RooGRS** [comment]
I can't install Windows 11 in version 2.9.1, or it doesn't identify the directories in data or documents, and when I can access at least the menu it says that something went wrong and I can't

**Reply by xoureldeen** [reply]
I dont understand, what the issue u face?

**Reply by Neverlose-sayooj** [reply]
> I dont understand, what the issue u face?

Bro the bootsrap downloading link is not working rebuild the link

**Reply by xoureldeen** [reply]
> > I dont understand, what the issue u face?
> 
> Bro the bootsrap downloading link is not working rebuild the link

use an biu version, Check README.md

**Comment by M1NGJ0** [comment]
When I am at the rom page it doesn't load the operating systems!

**Comment by nitin-121** [comment]
Do anyone have Bootstrap available offline ? Not possible to download

**Reply by meowsnow23** [reply]
i used the wayback machine and it works. heres the link
https://web.archive.org/web/20240619144950/https://vectrasvm.blackstorm.cc/vectras-vm/Releases/2.9.1/packages/vectras-vm-arm64-v8a.tar.gz

**Reply by xoureldeen** [reply]
Wht da, i wanna sue archive.org for this 😭

**Reply by se2crid** [reply]
Why do you want to sue?

**Reply by xoureldeen** [reply]
> Why do you want to sue?

Nvm its a joke.

**Reply by xoureldeen** [reply]
> Why do you want to sue?

The app updated and now it works fine

**Reply by meowsnow23** [reply]
but the app still not working. not downloading the bootstrap

**Reply by xoureldeen** [reply]
> but the app still not working. not downloading the bootstrap

Redownload and check ur internet

**Reply by meowsnow23** [reply]
why is it not worked
![Screenshot 2024-08-30 132409](https://github.com/user-attachments/assets/cf480e66-e672-4fcc-9c5f-5a98ce128d48)

**Reply by meowsnow23** [reply]
@xoureldeen

**Reply by meowsnow23** [reply]
or its just my internet

**Reply by xoureldeen** [reply]
we have server maintenance use this version

https://github.com/AnBui2004/Vectras-VM-Android

**Reply by Osobear14** [reply]
Where can I install vectras VM version 2.8?

**Reply by xoureldeen** [reply]
> Where can I install vectras VM version 2.8?

it's outdated you can't

**Comment by ghost** [comment]
**External mouse not working**

I'm using a Xiaomi Pad 6 Max which is running on Android 14.  I could not get the external mouse to work.  Tried adding "-usb -device usb-tablet" and "-usbdevice tablet" but couldn't work.  My Vectra is the latest version v2.9.1 (fix).

Any help is very much appreciated.

**Reply by xoureldeen** [reply]
Press options icon when u run the vm then u will see this option + may have bugs on some devices.

**Reply by ghost** [reply]
I could not see option or + icon.  :(

**Reply by xoureldeen** [reply]
> I could not see option or + icon.  :(

No, i mean Gear iconnn ⚙️

**Reply by ghost** [reply]
There's no + when I clicked on the Gear icon.

**Reply by xoureldeen** [reply]
> There's no + when I clicked on the Gear icon.  

R u dumb?, read the options in gear icon

**Reply by ghost** [reply]
That's so rude of you.  If I can see the options u mentioned I won't be asking for help..  Iy's alright.  Thanks for the crap.

**Comment by alwayssmilenj** [comment]
It asking for bootstrap bootstrap-x86_64-arm64-v8a.tar bit I can't find it pls help

**Comment by sleepeyme** [comment]
The website namee is boxvidra.blackstorm.cc not boxvidra.blacksorm.cc

**Reply by sleepeyme** [reply]
Go to here https://boxvidra.blackstorm.cc/

**Comment by Coolboy42-89** [comment]
Excuse me . I have a problem
![Screenshot_20240805-080028](https://github.com/user-attachments/assets/a16411e7-da62-42a9-8398-54d45ce9b2a8)
Always I get this error after setup

**Reply by xoureldeen** [reply]
> Excuse me . I have a problem
> ![Screenshot_20240805-080028](https://github.com/user-attachments/assets/a16411e7-da62-42a9-8398-54d45ce9b2a8)
> Always I get this error after setup
> 

Redownload and reinstall everything

**Comment by xxlm47** [comment]
It's not able to download the bootstrap not use the 1 I have

**Comment by helloworldpxy** [comment]
Omg 😱!why I can't download vectras-vm-arm64-v8a.tar.gz .There may be an issue with the server
![Screenshot_2024-09-09-07-44-04-16_5eea71194e1a68a05fef1cb2e772635c](https://github.com/user-attachments/assets/7cc921ac-9a28-4117-8cd1-0986342b628c)

**Comment by ilyaijn85** [comment]
It seems the host and Domain is down

**Comment by Isaaacehie** [comment]
![Screenshot_20241017_142922_Vectras VM](https://github.com/user-attachments/assets/fcb839ba-1f9d-4b4f-be3d-927a2484a28c)
How to fix

**Reply by xoureldeen** [reply]
use an biu version from readme.md

**Reply by Isaaacehie** [reply]
How im on mobile

**Reply by Armond452** [reply]
Try it by the bootstrap

**Comment by rlokeshk96** [comment]
![Screenshot_2024-10-22-00-28-35-02_5eea71194e1a68a05fef1cb2e772635c](https://github.com/user-attachments/assets/a5fba99d-1b5b-4eb7-acda-afbe753093bf)
Host and domain is down. Unable to download Bootstrap, please provide the alternate link to download them.

**Reply by xoureldeen** [reply]
use an bui, check the readme

**Reply by d4vdev** [reply]
How (I am dumb)

**Comment by d4vdev** [comment]
I don't know if this is a bug.

**Problem:**
When you add an iso file that the name of it has a quote (') it will give you an error and will not boot. 
, reputting a new iso file not help it either because I looked at the log. I found out that it will always intentionally add a quote to the end of the file path. Making it run into an error loop

(The error is undertamined quote something I forgot)

**How to fix by yourself**
Delete the old machine where the error occurs and make a new machine (also rename your iso file if it have a quote).

_i don't know much but that's what I found_

Have a good day.

**Comment by Armond452** [comment]
I have problem with this one I don't have the bootstrap 
![Screenshot_20241225_225751_Vectras VM](https://github.com/user-attachments/assets/5446620f-e385-44d2-a188-8c1d2f1414e9)
And this one 
![Screenshot_20241225_230007_Vectras VM](https://github.com/user-attachments/assets/490afcd7-5018-44de-9b07-66e6620a478e)

**Reply by Armond452** [reply]
And yes I only have a Chromebook

**Reply by xoureldeen** [reply]
> And yes I only have a Chromebook 

use the latest version.

**Comment by Armond452** [comment]
I have solved the problem try by the bootstrap  
It works for me if it doesn't work  root your phone

**Comment by Armond452** [comment]
i have rebuild vectras.netlily.app to rebuildvectrasvm-cloudflare2.netlify.app

**Reply by Armond452** [reply]
heres infromation 
![Screenshot 2025-02-04 9 33 39 AM](https://github.com/user-attachments/assets/86f808c1-536d-4d56-bbf3-bc49a3eb5636)
